build/common.ps1

```diff
@@ -492,10 +573,23 @@ Function Invoke-ILMerge {
         [string]$Configuration = $DefaultConfiguration
     )
     $nugetArtifactsFolder = Join-Path $Artifacts "$Configuration\NuGet.Clients\NuGet.CommandLine"
-    pushd $nugetArtifactsFolder
+    $dlls = Get-ChildItem $nugetArtifactsFolder -Filter 'nuget.*.dll' | %{ $_.Name }
```

Comment:
@emgarten It collects dlls from the project's output folder NuGet.CommandLine therefore only relevant assemblies produced by build are in.